test/index.js

```diff
@@ -301,7 +312,7 @@ test('NSpell()', function (t) {
 
     st.deepEqual(
       us.suggest('collor'),
-      ['color', 'collar', 'colloq'],
+      ['color', 'colloq', 'collar'],
       'should suggest removals'
```

Comment:
Here we favor "colloq" because "r" and "q" share a keyboard group while "a" and "o" do not.